File: Modul_6/Soal_5/PRAK605-2310817210015-MuhammadBukhariFitri.py
n = int(input())

matriksA = []
matriksB = []
matriksAXB = []

print("Matriks A")
for i in range (n):
    isi = list(map(int, input().split()))
    matriksA.append(isi)

print("Matriks B")
for i in range (n):
    isi = list(map(int, input().split()))
    matriksB.append(isi)

print("Matriks AXB")
for i in range(n):
   matriksAXB.append([])
   for j in range(n):
       a = 0
       for k in range(n):
          a += matriksA[i][k] * matriksB[k][j]
       matriksAXB[i].append(a)  

for isi in matriksAXB:
    for i in isi:
        print(f"{i} ", end = "")
    print()

File: Modul_6/Soal_5/PRAK605-2310817210015-MuhammadBukhariFitri.c
#include<stdio.h>
int main() {
    int n;
    scanf("%d", &n);
    int matriksA[n][n], matriksB[n][n], matriksAXB[n][n];
    printf("Matriks A\n");
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            scanf("%d", &matriksA[i][j]);
        }
    }
    printf("Matriks B\n");
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            scanf("%d", &matriksB[i][j]);
        }
    }
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            matriksAXB[i][j] = 0;
            for(int k = 0; k < n; k++) {
                matriksAXB[i][j] += matriksA[i][k] * matriksB[k][j];
            }
        }
    }
    printf("Matriks AXB\n");
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            printf("%d ", matriksAXB[i][j]);
        }
        printf("\n");
    }
}